### PR TITLE
Nt ecrs tasks index

### PR DESCRIPTION
**Pull request by nooralnabi**
add tasks  and list for transactions 
and add html  index for mail
and button to send email

### CODE REVIEW

**Comment by aqattan** [comment]
@nooralnabi Please fix the following issues:
1- When i click on any date in Listing Ecrs Tasks and i send an email, I got all transactions which i suppose to get only transactions of that specific day. 
2- Add a notification message to tell that email is sent successfully.
3- Add the environment to the email, so the receiver will know from which environment this email sent.

File: app/views/ecrs_mailer/ecrs_email.html.erb [hunk 1]
```diff
@@ -1,13 +1,84 @@
-<!DOCTYPE html>
+
```
Comment (ssharba) [inline app/views/ecrs_mailer/ecrs_email.html.erb]: @nooralnabi we are using haml in the application

File: config/environments/development.rb [hunk 2]
```diff
@@ -64,6 +64,6 @@
      :authentication       => "plain",
     :enable_starttls_auto => true
     }
-
+config.action_mailer.asset_host = "http://localhost:3000"
```
Comment (ssharba) [inline config/environments/development.rb]: what is this for?

File: app/views/ecrs_mailer/ecrs_email.html.erb (hunk 1 above)
Comment (nooralnabi) [inline app/views/ecrs_mailer/ecrs_email.html.erb]: @ssharba i think email templates it work with html it not support  haml

Comment (nooralnabi) [inline app/views/ecrs_mailer/ecrs_email.html.erb]: @ssharba i think email templates it work with html it not support  haml

Comment (ssharba) [inline app/views/ecrs_mailer/ecrs_email.html.erb]: Everything haml

**Pull request merged**